cpp/rat.cpp

```diff
@@ -333,6 +332,7 @@ RAT::Controls createControlsStruct(const Control& control)
     control_struct.resampleNPoints = control.resampleNPoints;
     stringToRatBoundedArray(control.boundHandling, control_struct.boundHandling.data, control_struct.boundHandling.size);
     control_struct.adaptPCR = control.adaptPCR;
+    control_struct.calcSLD = control.calcSLD;
```

Comment:
I don't think there is any need to expose `calcSLD` to the python code.

This should work fine as `control_struct.calcSLD = false;`